src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md

```diff
@@ -308,6 +308,16 @@ Successful: 1993 files; errors: 0
 ---
 ```
 
+The following command deploy only js with 4 jobs, deployment strategy is standard
+```bash
+bin/magento setup:static-content:deploy -s standard --no-misc --no-html --no-fonts --no-images --no-less --no-css -j 4
+```
+
+The following command deploy only css and less with 3 jobs, deployment strategy is quick
```

Comment:
```suggestion
The following command deploys only CSS and LESS with 3 jobs, deployment strategy is quick
```

Reply:
Hello @BarnyShergold 
Thank you for the suggestion, I have updated PR, can you review it?